modular_skyrat/modules/company_imports/code/armament_datums/jarnsmiour.dm

```diff
@@ -0,0 +1,94 @@
+/datum/armament_entry/company_import/blacksteel
+	category = BLACKSTEEL_FOUNDATION_NAME
+	company_bitflag = CARGO_COMPANY_BLACKSTEEL
+
+// A collection of melee weapons fitting the company's more exotic feeling weapon selection
+
+/datum/armament_entry/company_import/blacksteel/blade
+	subcategory = "Bladed Weapons"
+
+/datum/armament_entry/company_import/blacksteel/blade/hunting_knife
+	item_type = /obj/item/knife/hunting
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE * 1
+
+/datum/armament_entry/company_import/blacksteel/blade/hatchet
```

Comment:
"A crude axe blade upon a short wooden handle."
Doesn't seem like it'd fit

Reply:
~~add it to NRI~~